keras/preprocessing/image.py

```diff
@@ -1586,8 +1586,8 @@ def flow_from_directory(
               in the generator. See [this script](
               https://gist.github.com/fchollet/0830affa1f7f19fd47b06d4cf89ed44d)
               for more details.
-            target_size: Tuple of integers `(height, width)`, defaults to `(256,
-              256)`. The dimensions to which all images found will be resized.
+            target_size: Tuple of integers `(height, width)`. The dimensions to
+             which all images found will be resized. Defaults to `(256,256)`.
```

Comment:
The indentation of this line doesn't seem right. You can temporarily use 2 spaces.

Reply:
Ok @haifeng-jin I updated everything in that `flow_from_directory` docstring to be 2-spaced and have `Defaults to` correctly set